pkg/agent/datapath/policyBridge.go

```diff
@@ -580,6 +591,9 @@ func (p *PolicyBridge) AddMicroSegmentRule(rule *EveroutePolicyRule, direction u
 			if err := ruleFlow.LoadField("nxm_nx_reg4", 0x20, openflow13.NewNXRange(0, 15)); err != nil {
 				return nil, err
 			}
+			if err := ruleFlow.LoadField("nxm_nx_xxreg0", 0x1, openflow13.NewNXRange(127, 127)); err != nil {
```

Comment:
the default drop flow also need load ctlabel[127] = 1

Reply:
all drop flows will exec this part